dask/dataframe/groupby.py

```diff
@@ -1972,3 +2000,33 @@ def _value_counts_aggregate(series_gb):
     to_concat = {k: v.groupby(level=1).sum() for k, v in series_gb}
     names = list(series_gb.obj.index.names)
     return pd.Series(pd.concat(to_concat, names=names))
+
+
+def _tail_meta(series_gb, **kwargs):
+    return series_gb.tail(**kwargs)
+
+
+def _tail_chunk(series_gb, **kwargs):
+    keys, groups = zip(*series_gb) if len(series_gb) else ((True,), (series_gb,))
+    return pd.concat(
+        [group.tail(**kwargs) for group in groups], keys=keys, names=["_tail"]
+    )
+
+
+def _tail_aggregate(series_gb, **kwargs):
+    return series_gb.tail(**kwargs).droplevel("_tail")
+
+
+def _head_meta(series_gb, **kwargs):
+    return series_gb.head(**kwargs)
```

Comment:
This can probably be replaced with `M.head`.  This pattern is common enough that we made `M` to handle it.